client/components/draft_input/draft_input_caption.coffee

```diff
@@ -0,0 +1,204 @@
+React = require 'react'
+window.global = window
+window.process = {env: {NODE_ENV: 'development'}}
+
+{ convertToRaw,
+  convertFromHTML,
+  CompositeDecorator,
+  ContentState,
+  Editor,
+  EditorState,
+  RichUtils,
+  Modifier,
+  getVisibleSelectionRect } = require 'draft-js'
+{ stateToHTML } = require 'draft-js-export-html'
+DraftDecorators = require './draft_decorators.coffee'
```

Comment:
Seems similar conceptually -- this is how they mandate adding custom block types, or add classes and inline styles, will be getting deeper into these when eventually replacing scribe's plugins.  
 https://facebook.github.io/draft-js/docs/advanced-topics-decorators.htm